KeyError: "None of ['Period'] are in the columns"
Traceback:
File "C:\Users\ttamarz\OneDrive - Bank Negara Malaysia\RLMS\QC_Dashboard\app.py", line 465, in <module>
    main()
    ~~~~^^
File "C:\Users\ttamarz\OneDrive - Bank Negara Malaysia\RLMS\QC_Dashboard\app.py", line 284, in main
    outlier_df = find_outliers(
        current_series, prior_series,
        mom_pct_thresh_pct, yoy_thresh_pct,
        abs_cutoff, iqr_mult
    )
File "C:\Users\ttamarz\OneDrive - Bank Negara Malaysia\RLMS\QC_Dashboard\app.py", line 159, in find_outliers
    return pd.DataFrame(outliers).set_index("Period")
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
File "C:\Users\ttamarz\AppData\Roaming\Python\Python313\site-packages\pandas\core\frame.py", line 6129, in set_index
    raise KeyError(f"None of {missing} are in the columns")
